cmd/dev/app/rule_type/rttst.go

```diff
@@ -132,15 +131,12 @@ func testCmdRun(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("error reading fragment from file: %w", err)
 	}
 
-	remediateStatus := db.NullRemediationStatusTypes{}
+	remediateStatus := db.RemediationStatusTypesSkipped
```

Comment:
A recurring theme with many of the code changes in this PR is that the old queries returned a lot more null types. Due to the schema of the new tables, and the fact that evaluation history is now inserted transactionally, there are far fewer nullable fields.

Reply:
So "skipped" now means actually-skipped, or "didn't report a status"?

Reply:
(I do like removing null values and replacing with sentinels where possible, I'm just wondering whether having `""` as a default/valid value would be a good way to handle this.)

Reply:
Keep in mind that this file implements the CLI command for testing profiles/rules. The use of "Skipped" as a sentinel value vs NULL is of little consequence since this command persist anything to the DB.

Reply:
Oh, I completely forgot that this bit existed.  Nevermind!